File: 02_ProgrammingFundamentals/_04_Methods/_02_Exercise/_03_CharactersInRange.java
package _02_ProgrammingFundamentals._04_Methods._02_Exercise;

import java.util.Scanner;

public class _03_CharactersInRange {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input values
        char charOne = scan.nextLine().charAt(0);
        char charTwo = scan.nextLine().charAt(0);

        // Create method that prints ASCII chars in a given range
        printThis(charOne, charTwo);

        scan.close();
    }

    private static void printThis(char charOne, char charTwo) {

        for (int i = Math.min(charOne, charTwo) + 1; i < Math.max(charOne, charTwo); i++) {
            System.out.print((char) i + " ");
        }
    }
}

File: 24_C#Fundamentals/_04_Methods/_02_Exercise/_03_CharactersInRange/_03_CharactersInRange.cs
using System;

public class _03_CharactersInRange
{
    public static void Main(string[] args)
    {
        // Read input values
        char charOne = Console.ReadLine()[0];
        char charTwo = Console.ReadLine()[0];

        // Create method that prints ASCII chars in a given range
        printThis(charOne, charTwo);
    }

    private static void printThis(char charOne, char charTwo)
    {
        for (int i = Math.Min(charOne, charTwo) + 1; i < Math.Max(charOne, charTwo); i++)
        {
            Console.Write((char)i + " ");
        }
    }
}
